### DISCUSSION TITLE
Home screen

### DISCUSSION THREAD

**Discussion by creylopez** [victorbalssa/abacus#178] [Bug]
Just installed the app and after authentication et al, the home screen shows a sort of banner with the accounts listing and hiding 75% of the screen.

**Comment by victorbalssa** [comment]
Hey @creylopez,

I will need more info to debug.

Device: 
OS: (Android or iOS version)
Abacus version:  (0.10.0)

Maybe a screenshot (you can hide the NetWorth by pressing it).

It should look to something like this:

![Screenshot_1699451898](https://github.com/victorbalssa/abacus/assets/12813321/7c86c679-d53a-4c8f-bd73-0eea28bb7b98)

**Comment by creylopez** [comment]
I sent my last email without text, sorry. The accounts listing is always visible.

**Reply by victorbalssa** [reply]
@creylopez can you reply directly here: https://github.com/victorbalssa/abacus/discussions/178? 

Seems like the GitHub email reply isn't working well atm.

I don't understand: "The accounts listing is always visible." 

When you scroll horizontally do you have access to `Categories` and `Budgets` tabs?

**Reply by creylopez** [reply]
Sorry about that, I was on the move and found it easier to answer the email.
<img width="375" alt="abacus-1" src="https://github.com/victorbalssa/abacus/assets/54448654/dae9009b-87d5-4c6c-9dd5-bba752cd216c">

This the home screen

the black tab with "Cuentas (accounts)" hides almost the whole screen. Is this the way it is? As you can see in my iPhone 11 the summary is almost hidden. If I scroll horizontally by swipping the accounts tab it shows Categories and Budgets.

Just wanted to confirm that this is the expected behavior. I see in your screenshot that the central part (summary?) is more visible than in mine.

Otherwise, the app works quite well.

Thanks a lot

**Reply by victorbalssa** [reply]
Thank you I will fix it 👍🏻

**Reply by victorbalssa** [reply]
@creylopez should be fixed in `0.10.2`.
